app/javascript/widget/views/Home.vue

```diff
@@ -154,6 +167,14 @@ export default {
       );
     },
   },
+  mounted() {
+    bus.$on('on-attachment-error', () => {
```

Comment:
I would suggest keeping this name as a constant in a separate file if possible so that we know about all the events we are emitting in one place.